docs/ko/reference/compat/array/sample.md

```diff
@@ -0,0 +1,43 @@
+# sample
```

Comment:
I think we might not need docs for `sample` here, because we actually already have a one ([link](https://es-toolkit.slash.page/reference/array/sample.html)).